wg-odhDevSecOps/charter.md

```diff
@@ -0,0 +1,32 @@
+# WG DevSecOps
```

Comment:
naming conventions. `wg-odhDevSecOps/charter.md`  is a mixture of dash and CamelCase. Suggest we stick to dashed naming

Reply:
updated